Brokerages/Exante/ExanteClientWrapper.cs

```diff
@@ -0,0 +1,232 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading;
+using CryptoExchange.Net.Objects;
+using Exante.Net;
+using Exante.Net.Enums;
+using Exante.Net.Objects;
+
+namespace QuantConnect.Brokerages.Exante
+{
+    /// <summary>
+    /// Wrapper class for a Exante REST API client
+    /// </summary>
+    public class ExanteClientWrapper
+    {
+        private readonly ExanteClient _client;
+        
+        /// <summary>Get Exante client for stream data</summary>
+        public ExanteStreamClient StreamClient { get; private set; }
+
+        /// <summary>
+        /// Creates instance of wrapper class for a Exante REST API client
+        /// </summary>
+        public ExanteClientWrapper(ExanteClientOptions clientOptions)
+        {
+            _client = new ExanteClient(clientOptions);
+            StreamClient = new ExanteStreamClient(clientOptions);
```

Comment:
Missing to implement `IDisposable` here and disposing of these, should be called by the brokerage implementation which should also implement `IDisposable` 👍

Reply:
Done.